pv_site_api/main.py

```diff
@@ -355,6 +361,57 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     return res
 
 
+async def get_inverters_helper(session, inverter_ids):
+    if int(os.environ["FAKE"]):
+        return make_fake_inverters()
+
+    client = session.query(ClientSQL).first()
+    assert client is not None
+
+    async with httpx.AsyncClient() as httpxClient:
+        headers = {"Enode-User-Id": client.client_uuid}
+        if not inverter_ids.length:
+            return None
+        inverters_raw = await asyncio.gather(
+            *[
+                httpxClient.get(
+                    f"https://enode-api.production.enode.io/inverters/{id}", headers=headers
+                )
+                for id in inverter_ids
+            ]
+        )
+        inverters = [InverterValues(**inverter_raw.json()) for inverter_raw in inverters_raw]
+
+    return Inverters(inverters)
+
+
+@app.get("/inverters")
+async def get_inverters(
+    session: Session = Depends(get_session),
+):
+    client = session.query(ClientSQL).first()
+    assert client is not None
+
+    async with httpx.AsyncClient() as httpxClient:
+        headers = {"Enode-User-Id": client.client_uuid}
+        r = await httpxClient.get(
+            "https://enode-api.production.enode.io/inverters", headers=headers
+        ).json()
+        inverter_ids = [str(value) for value in r]
+
+    return get_inverters_helper(session, inverter_ids)
+
+
+@app.get("/sites/{site_uuid}/inverters")
+async def get_inverters_by_site(
+    site_uuid: str,
+    session: Session = Depends(get_session),
+):
+    inverter_ids = [inverter.inverter_uuid for inverter in _get_inverters_by_site(site_uuid)]
+
+    return get_inverters_helper(session, inverter_ids)
```

Comment:
Async functions need to be awaited to get their actual result.
```suggestion
    return await get_inverters_helper(session, inverter_ids)
```